### PR TITLE
ENH: Update CI for ITK 5.2rc03

### PR DESCRIPTION
**Pull request by tbirdso**

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -208,7 +208,7 @@ jobs:
       matrix:
         python-version-minor: [6, 7, 8, 9]
         include:
-          - itk-python-git-tag: "v5.1.2"
+          - itk-python-git-tag: "v5.2rc03"
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:211]: Probably also update `setup.py` to ref 5.2rc3 and increase version number.

**Comment by tbirdso** [comment]
Updated ITK commit tags to `7548a390d71dc25b80e792703ee3cfea17853a99` due to blocking build issue in `5.2rc03` tagged commit.

**Review by dzenanz — APPROVED** [review]

File: .github/workflows/build-test-package.yml [hunk 2]
```diff
@@ -136,7 +136,7 @@ jobs:
       matrix:
         python-version: [36, 37, 38, 39]
         include:
-          - itk-python-git-tag: "v5.1.2"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:139]: Tagged releases, i.e. `v5.2rc03` are only available for the Python package builds, but this is only what is required.

File: .github/workflows/build-test-package.yml [hunk 3]
```diff
@@ -172,7 +172,7 @@ jobs:
       max-parallel: 2
       matrix:
         include:
-          - itk-python-git-tag: "v5.1.2"
+          - itk-python-git-tag: "7548a390d71dc25b80e792703ee3cfea17853a99"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:175]: Tagged releases, i.e. `v5.2rc03` are only available for the Python package builds, but this is only what is required.

File: .github/workflows/build-test-package.yml [hunk 4]
```diff
@@ -208,7 +208,7 @@ jobs:
       matrix:
         python-version-minor: [6, 7, 8, 9]
         include:
-          - itk-python-git-tag: "v5.1.2"
+          - itk-python-git-tag: "7548a390d71dc25b80e792703ee3cfea17853a99"
```
Comment (thewtex) [inline .github/workflows/build-test-package.yml:211]: Tagged releases, i.e. `v5.2rc03` are only available for the Python package builds, but this is only what is required.

**Review by thewtex — CHANGES_REQUESTED** [review]
`PyVectorContainer` has been integrated into upstream ITK, so:

https://github.com/InsightSoftwareConsortium/ITKMeshToPolyData/blob/master/wrapping/PyVectorContainerBuffer.i.in
https://github.com/InsightSoftwareConsortium/ITKMeshToPolyData/blob/master/wrapping/itkPyVectorContainer.wrap
https://github.com/InsightSoftwareConsortium/ITKMeshToPolyData/blob/c79e2182934724ccb79a53ca8b5acd863fb99d93/wrapping/PyBuffer.i.init
https://github.com/InsightSoftwareConsortium/ITKMeshToPolyData/blob/c79e2182934724ccb79a53ca8b5acd863fb99d93/wrapping/CMakeLists.txt#L3

should also be removed.

**Review by thewtex — APPROVED** [review]

**Comment by thewtex** [comment]
@tbirdso thanks!

**Pull request merged**